src/dagent/src/routers/m9k/api/caller/array.go

```diff
@@ -142,7 +142,9 @@ func CallListArray(xrId string, param interface{}, posMngr pos.POSManager) (mode
 		log.Errorf(commandFailureMsg, GetFuncName(1), err1)
 		return model.Response{}, ErrConn
 	}
-	resByte, err2 := protojson.Marshal(result)
+	resByte, err2 := protojson.MarshalOptions{
+		EmitUnpopulated: true,
+	}.Marshal(result)
 	return HandleResponse(resByte, err2)
```

Comment:
what is the purpose of below here ?
MarshalOptions{
		EmitUnpopulated: true,
	}.Marshal(result)

Reply:
If this is not added, then 0 values, empty strings and null values will not be present in the JSON output